service/src/main/java/uk/gov/hmcts/reform/fpl/model/Children.java

```diff
@@ -8,18 +8,22 @@
 import lombok.Builder;
 import lombok.Data;
 import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasChildName;
 
 import java.util.Arrays;
 import java.util.List;
 import java.util.UUID;
 import java.util.stream.Collectors;
+import javax.validation.Valid;
 
 @Data
 @Builder
 @JsonIgnoreProperties(ignoreUnknown = true)
+@HasChildName
```

Comment:
Why custom annotation?

Looks like annotating collection with `@Size(min=1)` and every element with `@Valid` annotation .